spec/lib/declare_schema/model/foreign_key_definition_spec.rb

```diff
@@ -60,6 +60,14 @@ class Network < ActiveRecord::Base
           expect(subject.on_delete_cascade).to be_truthy
         end
       end
+
+      context 'when constraint name passed' do
+        let(:options) { { parent_table: :networks, foreign_key: :the_network_id, index_name: :index_on_network_id,
```

Comment:
I thought we were getting rid of the `index_name:` option? I think we only need `constraint_name:`.

Reply:
Great point, there's no need for the index_name option anymore!